ding/reward_model/gail_irl_model.py

```diff
@@ -113,35 +112,50 @@ class GailRewardModel(BaseRewardModel):
+        1  ``type``              str        gail           | RL policy register name, refer    | this arg is optional,
+                                                           | to registry ``POLICY_REGISTRY``   | a placeholder
+        2  | ``expert_data_``    str        expert_data.   | Path to the expert dataset        | Should be a '.pkl'
+           | ``path``                       .pkl           |                                   | file
+        3  | ``learning_rate``   float      0.001          | The step size of gradient descent |
+        4  | ``update_per_``     int        100            | Number of updates per collect     |
+           | ``collect``                                   |                                   |
+        5  | ``batch_size``      int        64             | Training batch size               |
+        6  | ``input_size``      int                       | Size of the input:                |
+           |                                               | obs_dim + act_dim                 |
+        7  | ``target_new_``     int        64             | Collect steps per iteration       |
+           | ``data_count``                                |                                   |
+        8  | ``hidden_size``     int        128            | Linear model hidden size          |
+        9  | ``collect_count``   int        100000         | Expert dataset size               | One entry is a (s,a)
+           |                                               |                                   | tuple
+        10 | ``clear_buffer_``   int        1              | clear buffer per fix iters        | make sure replay
+           | ``per_iters``                                                                     | buffer's data count
+           |                                                                                   | isn't too few.
+           |                                                                                   | (code work in entry)
+        == ====================  ========   =============  =================================== =======================
+    """
     config = dict(
+        # (str) RL policy register name, refer to registry ``POLICY_REGISTRY``.
         type='gail',
+        # (float) The step size of gradient descent.
         learning_rate=1e-3,
+        # (int) How many updates(iterations) to train after collector's one collection.
+        # Bigger "update_per_collect" means bigger off-policy.
+        # collect data -> update policy-> collect data -> ...
         update_per_collect=100,
+        # (int) How many samples in a training batch.
         batch_size=64,
+        # (int) Size of the input: obs_dim + act_dim
         input_size=4,
+        # (int) Collect steps per iteration
         target_new_data_count=64,
+        # (int) Linear model hidden size
```

Comment:
why linear here, there maybe some more complicated networks

Reply:
because our remodel network is linear, should we change this comment?